post.hbs

```diff
@@ -81,6 +81,60 @@
 
 {{#get "posts" limit="3" include="tags,authors" filter="id:-{{id}}+primary_tag:{{primary_tag.slug}}"}}
     {{#if posts}}
+        <div class="download-status">
+            <div class="row">
+                <div class="ml-auto mr-auto">
+                    <h3>{{t "Download Status for"}}</h3>
+                    <div class="download-container">
+                        <div class="download-box">
+                            <div class="download-copy-container">
+                                <div class="download-title">Mobile</div>
+                                <div class="download-description">Private messaging, secure crypto transactions, Web3 DApp browsing in your pocket.</div>
+                                <div class="download-buttons">
+                                    <a href="https://apps.apple.com/us/app/status-private-communication/id1178893006" id="official:ios" class="">
+                                        <img class="mobile-download" src="{{asset "img/app-store.svg"}}">
+                                    </a>
+                                    <a href="https://play.google.com/store/apps/details?id=im.status.ethereum" id="official:android" class="download-margin-left">
+                                        <img class="mobile-download" src="{{asset "img/google-play.svg"}}">
+                                    </a>
+                                    <a href="https://status-im-files.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-v1.10.1.apk" id="official:apk" class="download-margin-left">
```

Comment:
Do we REALLY want YET ANOTHER place to keep updating whenever releases change?

Reply:
@jakubgs I thought of this as well. We should have static links for the `apk` and `desktop builds` that link to the latest version. Do you have any solution in mind?

Reply:
Yeah, I need to think a bit about that. How to do it properly so we have one place to update this stuff.

Reply:
I haven't forgotten about this.